.babelrc

```diff
@@ -1,9 +1,17 @@
 {
-  "presets": ["es2015"],
-  "plugins": [
-    "transform-flow-strip-types",
-    "transform-es3-property-literals",
-    "transform-es3-member-expression-literals",
-    ["transform-es2015-for-of", {"loose": true}]
```

Comment:
I am adding back all of your babel transforms.  I tried to take a minimalistic approach to get rid of things that aren't needed.  If you think they are needed then I'm good with that.  Its odd though that there are no errors on Phantom for the for/of since phantom really isn't ES6y.